node/src/stream_handler_pool.rs

```diff
@@ -300,7 +300,7 @@ impl StreamHandlerPool {
                 };
                 debug!(
                     self.logger,
-                    "Sending node query about {:?} to Neighborhood", key
+                    "Sending node query about {} to Neighborhood", key
```

Comment:
It turns out the key is printed the same way using `Display` or `Debug`.

This is an example of the log:


```bash
DEBUG: Dispatcher: Sending node query about 59uGFlLht3firKunA64sRKUv9Itgt6AKsNQa/ZYOTgU to Neighborhood
```

Since, it's a Debug log, I think we should use `{:?}` instead of `{}`, to keep the convention.

Also, there wasn't any assertion for this debug log inside the tests that were testing this section of code. So, I introduced one.